Misc/NEWS.d/next/Library/2020-05-02-04-29-31.bpo-40459.fSAYVD.rst

```diff
@@ -0,0 +1 @@
+Fix :exc:`NameError` caused from :func:`platform.win32_ver`.
```

Comment:
The NEWS entry doesn't really explain the behavior change for users.

It seems like previously, ptype was always an empty string. ptype is now filled with the value of the "SOFTWARE\Microsoft\Windows NT\CurrentVersion" registry entry.

Can someone test on Windows to see how this string look like? I'm curious :-)

The documentation says:

> As a hint: ptype is 'Uniprocessor Free' on single processor NT machines and 'Multiprocessor Free' on multi processor machines. The ‘Free’ refers to the OS version being free of debugging code. It could also state ‘Checked’ which means the OS version uses debugging code, i.e. code that checks arguments, ranges, etc.

https://docs.python.org/dev/library/platform.html#platform.win32_ver

Reply:
Please explain that this change fix the ptype part of :func:`platform.win32_ver` result.